src/components/Login/LoginContainer.tsx

```diff
@@ -115,7 +123,15 @@ export const InitialisationContainer: FunctionComponent<NavigationProps> = ({
     if (token && endpoint && !features) {
       setEnvVersion();
     }
-  }, [endpoint, token, setFeatures, setProducts, features]);
+  }, [
+    endpoint,
+    token,
+    setFeatures,
+    setProducts,
+    features,
+    handleLogout,
+    expiry
```

Comment:
Don't need this dependency i think

Reply:
oops thanks! also removed navigation attribute from loginOTPcard